classjugaad.py

```diff
@@ -0,0 +1,50 @@
+# class jugaad 
+import re
+import os
+import fileinput
+import random
+def classJugaad(contents):

```

Comment:
this is a function, rename the function to something more appropriate and avoid using class, since you know, it's a function.